docs/examples/particles/lib/main.dart

```diff
@@ -227,6 +229,23 @@ class MyGame extends BaseGame {
     );
   }
 
+  /// Same example as above, but using awesome [Inverval]
+  /// curve, which "schedules" transition to happen between
+  /// certain values of progress. In this example, circles will
+  /// move from their initial to their final position
+  /// when progress is changing from 0.2 to 0.6 respectively.
+  Particle constantVelocityMovingParticle() {
```

Comment:
1. s/Inverval/Interval/
2. I don't see you using the Interval curve here. In fact this looks very weird to me because you say constant velocity but that you are using a curve? what do you use a curve for if the velocity is constant?